charts/ndb-operator/templates/deployment-ndb-operator-controller-manager.yaml

```diff
@@ -2,77 +2,87 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    control-plane: {{ include "ndb-operator.fullname" . }}-controller-manager
+    control-plane: controller-manager
   name: {{ include "ndb-operator.fullname" . }}-controller-manager
   namespace: {{.Release.Namespace}}
 spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      control-plane: {{ include "ndb-operator.fullname" . }}-controller-manager
+      control-plane: controller-manager
```

Comment:
These are labels, and not resource names. I believe we shouldn't templatize them. Removing the old templating logic here. Please confirm @tuxtof